pennylane/compiler/__init__.py

```diff
@@ -56,6 +57,15 @@
     ~compiler.active_compiler
     ~compiler.active
 
+Presented below is the list of :func:`~.qjit` compatible
+PennyLane primitives.
+
+.. autosummary::
+    :toctree: api
+
+    ~grad
+    ~jacobian
```

Comment:
@josh146 what are your opinions on the addition of this list?

Reply:
I like it! Good idea to include this.